extensions/vscode-api-tests/src/singlefolder-tests/interactiveWindow.test.ts

```diff
@@ -35,7 +35,9 @@ async function addCell(code: string, notebook: vscode.NotebookDocument) {
 
 async function addCellAndRun(code: string, notebook: vscode.NotebookDocument, i: number) {
 	const cell = await addCell(code, notebook);
+	const event = asPromise(vscode.workspace.onDidChangeNotebookDocument);
```

Comment:
Would it stil be flaky if the execute doesn't succesfully happen?

Reply:
If the execute fails, then the test _should_ fail, right?

Reply:
> If the execute fails, then the test should fail, right?

@roblourens it should fail but it's not a bug as without notebook document uri passed in, `notebook.cell.execute` would try to find the active notebook editor, which can be falky as it's focus based.